### ISSUE TITLE
Create a generic "Effect Amount" adjustment for selected effect

### ISSUE THREAD

**Issue by JennyMeissner** [CommandPost/LoupedeckPlugin#13]
Can we create a generic "Effect Amount" adjustment so that the Amount slider of the effect that is currently selected, gets adjusted? Possibly also a "Effect Brightness" adjustment that works whenever a selected effect has a Brightness slider.

**Comment by latenitefilms** [comment]
I'm not really sure how that would work. However, one idea could be to have an action that controls whatever slider is currently under the mouse?

**Comment by latenitefilms** [comment]
Having a knob control for "Amount" is tricky because the only way we can really detect which is the active effect is by visually looking for the yellow or white outline - there's nothing in the Accessibility API that allows us to determine which effect is active.

We also currently don't have any CommandPost APIs for individual effects, so this is something we'd need to build, which could also be tricky, just because of how things are organised in the Accessibility tree.

Long story short - I don't think this is something we can easily add quickly - it might have to come later down the line.

<img width="426" alt="image" src="https://user-images.githubusercontent.com/22286696/169760370-7f48e29b-17e8-4ce5-88c6-f89646bfdaa0.png">

<img width="408" alt="image" src="https://user-images.githubusercontent.com/22286696/169760420-31c9a16e-df29-409c-af9a-4b50c252427d.png">

**Comment by latenitefilms** [comment]
I'm going to close this for now, as the "Nudge Up/Down" keyboard shortcuts kinda do the same thing (for the selected parameter). If we ever add individual effects to the CommandPost FCPX API, we can go back and look at this for Loupedeck.
